inspirehep/modules/hal/utils.py

```diff
@@ -458,16 +491,7 @@ def get_page_artid(record):
 
     """
     publication_info = get_value(record, 'publication_info[0]', default={})
-
-    if 'artid' in publication_info:
-        artid = publication_info['artid']
-        return artid
-    elif 'page_start' in publication_info and 'page_end' in publication_info:
-        page_start = publication_info['page_start']
-        page_end = publication_info['page_end']
-        return '{}-{}'.format(page_start, page_end)
-
-    return ''
+    return get_page_artid_for_publication_info(publication_info, separator)
```

Comment:
I dislike all the changes in this file, as they:
1. Break the consistency of the previous API, which previously had only functions of type `record -> list(str)` or `record -> str`.
1. Modify code that was tested on 4000+ records during the HAL exports.

So please don't depend anywhere else on these utils, as they are going to be refactored ASAP!